src/pages/lessons/[slug].tsx

```diff
@@ -247,8 +195,6 @@ const Lesson: React.FC<LessonProps> = ({
   const transcriptAvailable = transcript || enhancedTranscript
   const {session_id} = router.query
 
-  const primary_tag = get(first(get(lesson, 'tags')), 'name', 'javascript')
-
```

Comment:
This line sat in the middle of a bunch of data setup, but is only ever used by one overlay. It now lives much closer to where it is being used.